### PR TITLE
Assignment 1 - Playlist which works with Non-explicit and explicit tracks

### PR DESCRIPTION
**Pull request by binalee21**

### CODE REVIEW

File: cypress/integration/createPlaylist.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import constant from "../fixtures/constants.json";
+
+describe('Create new playlist', () => {
+
+    it('Create Playlist which works with Non-explicit and explicit tracks', () => {
+        //click on new playlist button
+        cy.get('button.ant-btn > .ng-star-inserted').click();
```
Comment (Shreya76) [inline cypress/integration/createPlaylist.js:7]: Please add assertions after every action.

File: cypress/integration/createPlaylist.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        //Select Editors value from autocomplete Textbox
+        cy.get('[nzerrortip="Editors is required"]').type(constant.editors);
+        cy.get('re-tag-content > span').click({ force: true });
+
+        //Entering Title, description and author
+        cy.get('#title').type(constant.playlistTitle);
+        cy.get('#description').type(constant.playlistDescription);
+        cy.get('#author').type(constant.playlistAuthor);
+
+        //image selection
```
Comment (Shreya76) [inline cypress/integration/createPlaylist.js:18]: Please create custom commands for methods which can be reused.

File: cypress/integration/createPlaylist.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            //Handling visibility of confirmation popup ok button
+            cy.get('body').then(($body) => {
+                const buttonConfirmation = $body.find('.ant-modal-confirm-btns > .ant-btn-primary');
+                if (buttonConfirmation.length) buttonConfirmation.click();
+            });        
+            cy.get('.ant-notification-notice-message').should('have.text', 'Save Success');
+            cy.get('.ant-notification-notice-close-x').click({ force: true });
+        }
+
+        //publish playlist and verify
```
Comment (Shreya76) [inline cypress/integration/createPlaylist.js:79]: create multiple test cases for different functionalities.

**Review by Shreya76 — CHANGES_REQUESTED** [review]

File: cypress/integration/createPlaylist.js (hunk 3 above)
Comment (binalee21) [reply cypress/integration/createPlaylist.js]: added more assertions, custom commands and break into multiple testcases.

**Pull request merged**